### PR TITLE
Language patches - Polish and Silesian

### PR DESCRIPTION
**Pull request by Daaaav**
## Changes:

Updates to Polish and Silesian were delivered a couple hours ago.

## Legal Stuff:

By submitting this pull request, I confirm that...

- [x] My changes may be used in a future commercial release of VVVVVV
- [x] I will be credited in a `CONTRIBUTORS` file and the "GitHub Friends"
  section of the credits for all of said releases, but will NOT be compensated
  for these changes

### CODE REVIEW

File: desktop_version/lang/pl/strings.xml [hunk 1]
```diff
@@ -502,16 +502,16 @@
...
-    <string english="Potential for Anything" translation="Potential for Anything" explanation="jukebox prompt. song name should probably not be translated" max="37*2"/>
-    <string english="Predestined Fate" translation="Predestined Fate" explanation="jukebox prompt. song name should probably not be translated" max="37*2"/>
-    <string english="Pipe Dream" translation="Pipe Dream" explanation="jukebox prompt. song name should probably not be translated" max="37*2"/>
-    <string english="Popular Potpourri" translation="Popular Potpourri" explanation="jukebox prompt. song name should probably not be translated" max="37*2"/>
-    <string english="Pressure Cooker" translation="Pressure Cooker" explanation="jukebox prompt. song name should probably not be translated" max="37*2"/>
-    <string english="ecroF evitisoP" translation="ecroF evitisoP" explanation="jukebox prompt. song name should probably not be translated" max="37*2"/>
+    <string english="Passion for Exploring" translation="Pasja do Eksploracji|(Passion for Exploring)" explanation="jukebox prompt. song name should probably not be translated" max="37*2"/>
```
Comment (InfoTeddy) [inline desktop_version/lang/pl/strings.xml:505]: The song name and subtitles seem to be inconsistent with the Russian translation. Whereas Russian puts English first and the translation in parentheses after, these put the translation first and English in parentheses after.

Comment (mothbeanie) [reply desktop_version/lang/pl/strings.xml:505]: English song name first, translation as subtitle seems ideal to me. we should probably favor the one we already have and change this one i think

Comment (Daaaav) [reply desktop_version/lang/pl/strings.xml:505]: I think it should be whatever's appropriate in each language - most languages don't translate the song names at all, while some translate them and remove the English ones completely. So maybe for Russian it's appropriate to keep them English but subtitle them, while for Polish and Silesian it's appropriate to put a translated one first and foremost - consistency across languages should not override that.

Maybe that's not the case here though, so let's ask the translator.

Comment (onetruepurple) [reply desktop_version/lang/pl/strings.xml:505]: Feel free to rearrange these as you see fit. (Will also need to be applied to the unlock prompts in cutscenes.xml)

Comment (Daaaav) [reply desktop_version/lang/pl/strings.xml]: Done and force-pushed.

**Review by InfoTeddy — APPROVED** [review]

**Pull request merged**